src/akkd/Message/Message.tsx

```diff
@@ -9,39 +9,45 @@ interface MessageTypes {
 	warning?: boolean
 	info?: boolean
 	success?: boolean
+	default?: boolean
 }
 
 const typePrecedence: ReadonlyArray<keyof MessageTypes> = [
 	'error',
 	'warning',
 	'info',
 	'success',
+	'default',
 ]
 
 interface Props extends MessageTypes {
+	as?: React.ReactType,
+	className?: string,
```

Comment:
I'm a little adverse to passing classNames in, though I recognise `Container` does it. is it something avoidable or does it bring significant benefit?

Reply:
IIRC, I'd passed classNames in to a few components because classNames were currently being passed through somewhere, but AFAIK there's no actual benefit.